src/ViewHoldingsRecord.js

```diff
@@ -846,7 +844,9 @@ ViewHoldingsRecord.propTypes = {
     temporaryLocationQuery: PropTypes.object.isRequired,
   }),
   onCloseViewHoldingsRecord: PropTypes.func.isRequired,
+  updateLocation: PropTypes.func,
+  goTo: PropTypes.func,
```

Comment:
Should these new props be required? I found the same situation in other files where that props added. Please check other places too.

Reply:
Thanks for catching this @VictorSoroka1!